### DISCUSSION TITLE
ML-KEM macros in ML-DSA header

### DISCUSSION THREAD

**Discussion by abhi-dev-engg** [open-quantum-safe/liboqs#1801] [General]
Hi all,

There are some macros - ciphertext , shared secret from KEM present in ML-DSA header file instead of signature length.
https://github.com/open-quantum-safe/liboqs/blob/main/src/sig/ml_dsa/sig_ml_dsa.h#L20

Also encaps,decaps declaration are present.

**Comment by SWilson4** [comment]
Hi @yogappu, thanks for raising this. We have an issue open (open-quantum-safe/liboqs#1797) and a fix (open-quantum-safe/liboqs#1799) that should be merged shortly.
